README.markdown

```diff
@@ -7793,10 +7793,38 @@ tcpsock:setoption
 
 **context:** *rewrite_by_lua&#42;, access_by_lua&#42;, content_by_lua&#42;, ngx.timer.&#42;, ssl_certificate_by_lua&#42;, ssl_session_fetch_by_lua&#42;*
 
-This function is added for [LuaSocket](http://w3.impa.br/~diego/software/luasocket/tcp.html) API compatibility and does nothing for now. Its functionality will be implemented in future.
+This function is added for [LuaSocket](http://w3.impa.br/~diego/software/luasocket/tcp.html) API compatibility and does nothing for now. Its functionality is implemented `v0.10.18`.
 
 This feature was first introduced in the `v0.5.0rc1` release.
 
+`option` is a string with the option name, and value depends on the option being set:
+
+* `keepalive`
+	Setting this option to true enables sending of keep-alive messages on
+	connection-oriented sockets(e.g., tcpsock:setoption("keepalive", 1) and
```

Comment:
You should render the code snippets as code. Fix other similar places.

Reply:
accept